src/search.c

```diff
@@ -40,6 +41,121 @@ bool init = false;
 static char search_type = '0';
 static char scope = '0';
 
+struct bundle_result {
+	char bundle_name[256];
+	long size;
+	double score;
+	struct list *files;
+};
+
+struct file_result {
+	char filename[PATH_MAX+1];
+	double score;
+};
+
+static struct list *results;
+
+
+static void add_bundle_file_result(char *bundlename, char *filename, double score, struct manifest *man) 
+{
+	struct bundle_result *bundle = NULL;
+	struct file_result *file;
+	struct list *ptr;
+
+	ptr = results;
+	while (ptr) {
+		struct bundle_result *item;
+
+		item = ptr->data;
+		if (strcmp(item->bundle_name, bundlename) == 0) {
+			bundle = item;
+			break;
+		}
+		ptr = ptr->next;
+	}
+	if (!bundle) {
+		bundle = calloc(sizeof(struct bundle_result), 1);
+		results = list_append_data(results, bundle);
+		strncpy(bundle->bundle_name, bundlename, 255);
+		bundle->size = man->contentsize / 1024 / 1204;
+		bundle->score = -0.1 * bundle->size;
```

Comment:
Some comments would be nice.

Reply:
Hints like why 1204 was chosen would be good as well, rather than making me suspect a typo for 1024.

Reply:
Btw, I believe 1204 is a typo in the MiB calculation. The score hit depending on bundle size might need a comment, yes. @fenrus75?

Reply:
Since it is setting an initial score of negative 1/10th the bundle size, it seems to just be an initial penalty based on the size of the bundle. I'll add a comment to that effect.

Reply:
Also, confirmed that 1204 is a typo for 1024